### DISCUSSION TITLE
Ubuntu version?

### DISCUSSION THREAD

**Discussion by KyleMaas** [mediacms-io/mediacms#661] [Q&A]
The docs mention Ubuntu 18 or 20:

https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md#2-server-installation

Has MediaCMS been tested on Ubuntu 22 yet?  Because with trying to get live streaming to work, a lot of the other projects require dependencies which are only available on 22 without manually installing them, which is complicating my efforts quite a bit.

**Comment by mgogoulos** [comment]
I did a successful installation on Ubuntu 22 recently but don't remember any specific trouble I got into - it should be another if/else switch on the server installation https://github.com/mediacms-io/mediacms/blob/main/install.sh

Regarding the Docker installation, I was thinking that this needs to be updated too, it is based on a Python3.8 official image (which is based on Debian) so that could be switched to Ubuntu 22 too...
